astropy/utils/iers/tests/test_leap_second.py

```diff
@@ -109,6 +109,11 @@ def test_auto_open_simple(self):
         ls = iers.LeapSeconds.auto_open([iers.IERS_LEAP_SECOND_FILE])
         assert ls.meta['data_url'] == iers.IERS_LEAP_SECOND_FILE
 
+    def test_auto_open_max_age_none(self):
+        with iers.conf.auto_max_age.set_temp(None):
```

Comment:
I'm not sure this test actually tests what you want. I think you'd want to create a file that is for sure out of date, and pass both that file and the real file in, i.e., `auto_open([outdated_file, iers.IERS_LEAP_SECOND_FILE])`, and then check that it loads `outdated_file`. It may be easiest to copy things from `test_fake_expired_file` (or even just to tag it on there).

Reply:
Good point.  I was only checking to make sure an exception was not thrown.  I'll take a look.

Reply:
See latest commit.